package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "bugzilla-to-webcompat",
+  "version": "1.0.0",
+  "description": "Adding a button to bugzilla ui allowing to move a bug to webcompat.com",
+  "main": "moveToWebcompat.js",
+  "scripts": {
+    "start": "npx web-ext run",
+    "lint:addon": "npx web-ext lint",
+    "prettier": "npx prettier --write '{*.css,*.js}'",
+    "lint:js": "npx eslint .",
+    "lint:js:fix": "npx eslint ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/webcompat/bugzilla-to-webcompat.git"
+  },
+  "author": "ksy36",
+  "license": "ISC",
+  "homepage": "https://github.com/webcompat/bugzilla-to-webcompat#readme",
+  "devDependencies": {
+    "eslint": "^7.13.0",
+    "prettier": "^2.1.2",
```

Comment:
As of right now, Mozilla projects generally are on `1.19.1`, which results in slightly different formatting in some cases. It's fine to stay on 2.x if you want, but I figured it's worth highlighting.